packages/biotk/biotk.0.0.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis: "Bioinformatics toolkit"
+description: """
+biotk implements various functionalities in
+                computational biology, like parsers/unparsers for some
+                file formats, a few basic data structures and sequence
+                algorithms, as well as access to some databases."""
+maintainer: ["philippe.veber@gmail.com"]
+authors: ["Philippe Veber"]
+license: "CeCILL-B"
+homepage: "https://github.com/pveber/biotk/"
+bug-reports: "https://github.com/pveber/biotk/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "angstrom" {>= "0.15.0"}
+  "angstrom-unix" {>= "0.15.0"}
+  "biocaml"
+  "core" {>= "v0.14.0"}
+  "fmt"
+  "ocaml" {>= "4.11.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.11.0" & < "4.13.0"}
```
Is the failure a bug in 4.13 or the package is actually incompatible with 4.13?

Reply:
Thanks to @kit-ty-kate, we now know this is a (fixed) bug in the compiler (see ocaml/ocaml#10529)